packages/dashboard/src/setupTests.ts

```diff
@@ -1,5 +1,6 @@
 import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
```

Comment:
From this PR and on,  Enzyme will no longer be used, we can remove the enzyme dependencies from `package.json`. And from here too.

Reply:
got it!

Reply:
hi @matiasbavera , for this current PR, is it alright to leave these import lines first? removing them will break the tests in the react tour component and I think it's more efficient to address that in a separate PR. there are several parts in the tour component that needs to be updated with the changes in the dashboard (eg. using a stack for the menu navigation)

Reply:
Yeah, you're right!